libgalois/src/PropertyGraph.cpp

```diff
@@ -243,8 +218,12 @@ katana::PropertyGraph::Copy(
     const std::vector<std::string>& node_properties,
     const std::vector<std::string>& edge_properties) {
   // TODO(gill): This should copy the RDG in memory without reloading from storage.
-  return Make(
-      rdg_dir(), partition_number(), &node_properties, &edge_properties);
+  tsuba::RDGLoadOptions opts;
+  opts.partition_to_load = partition_number();
+  opts.node_props = &node_properties;
+  opts.edge_props = &edge_properties;
+
+  return Make(rdg_dir(), opts);
```

Comment:
I think you can write `Make(rdg_dir(), {.partition_to_load = partition_number(), .node_props = &node_properties,
.edge_props = &edge_properties});` which at least to me is clearer.

Reply:
As @ddn0 said below. I'm wrong about this.